spec.md

```diff
@@ -523,8 +547,75 @@ schema being used.
         "bullets": [
           "40 concurrent connections"
         ]
+      },
+      "schemas": {
+        "service_instance": {
+          "create": {
+            "parameters": {
+              "$schema": "http://json-schema.org/draft-04/schema#",
+              "$ref": "/v2/catalog/root/#BillingAccountParameters"
```

Comment:
Does root_schemas needs to be an array?
Would it be equivalent to have a map and keep using the standard [$ref syntax](https://spacetelescope.github.io/understanding-json-schema/structuring.html) 

definitions:
  BillingAccountParameters:
bindingDefinitions
  BillingAccountParameters

So that you can use

$ref: #/definitions/BillingAccountParameters
$ref: #/bindingDefinitions/BillingAccountParameters

Reply:
It is less complicated to have the root_schemas object as an array for aggregation/proxy brokers.

And you can't use `#/yada/yada` in the catalog schemas because those reference the local document schema for that catalog plan. It needs the full path that comes from the `$id` of the root schema, and the `$ref` has to be an external reference.

Reply:
the root schema $id is meant to be a full uri but is not required. So I am taking advantage of the external $ref definition from the json schema spec, but putting a constraint that you need to provide that root schema in the catalog. In future versions of the OSB I would like to revisit this limitation, and allow fully external schemas. But At the F2F it was decided that fully external schemas was too much risk at the moment. So we have this half step.

To be clear, I am not inventing anything, I am just using json schemas mechanism for root schema $id links. I am adding a convention that platforms should process the root schemas array first to be able to know about links used in the plans schemas object.